engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -262,28 +230,49 @@ case List(expr) => Some(expr)
         var translatedConstructors = CollectionConverters.asScala(translatedConstructorsIt).toList();
         // type
         List<IR.DefinitionArgument> args = translateArgumentsDefinition(def.getParams());
-        yield new IR$Module$Scope$Definition$SugaredType(
+        var t = new IR$Module$Scope$Definition$SugaredType(
           typeName,
           args,
           translatedConstructors.appendedAll(translatedBody),
           getIdentifiedLocation(inputAst),
           meta(), diag()
         );
+        yield cons(t, appendTo);
       }
       case Tree.Function fn -> {
         var methodRef = translateMethodReference(fn.getName(), false);
         var args = translateArgumentsDefinition(fn.getArgs());
         var body = translateExpression(fn.getBody(), false);
 
-        yield new IR$Module$Scope$Definition$Method$Binding(
+        var b = new IR$Module$Scope$Definition$Method$Binding(
```

Comment:
it's generally easier to understand vars if they are named with something more than a single letter :)

Reply:
OK. All single letter variables renamed in [207583e](https://github.com/enso-org/enso/pull/3806/commits/207583e11df6f84e816af2feed0d6d384ef1c42b)